donkeycarManagerFront/src/components/register.vue

```diff
@@ -80,33 +80,23 @@ export default {
       this.$router.push('/')
     },
     async addUser () {
-      this.playerTest = await srv.getPlayerByPseudo(this.pseudo)
-      console.log(this.playerTest)
-      console.log(this.playerTest.length)
-      if (this.playerTest.length === 0){
-        console.log('pass')
-        this.player = await srv.createPlayer(this.pseudo)
-        console.log('pass1.1')
-        await srv.addJobs(this.player.player_id)
-        console.log('pass2')
-        if (this.waitingList.length < 2) {
-          this.attente = 'Maintenant'
-        } else if (this.waitingList.length >= 2 && this.waitingList.length < 4) {
-          this.attente = '15 minutes'
-        } else if (this.waitingList.length >= 4 && this.waitingList.length < 8) {
-          this.attente = '30 minutes'
-        } else if (this.waitingList.length >= 8 && this.waitingList.length < 12) {
-          this.attente = '45 minutes'
-        } else if (this.waitingList.length >= 12) {
-          this.attente = 'plus de 1 heure'
-        }
-        this.numero = this.player.player_id
-        this.popup = true
-        console.log('pass3')
-      }else if (this.playerTest.length !== 0){
-        this.popup404 = true
+      const players = await srv.getPlayerByPseudo(this.pseudo)
+      if (players.length === 0) {
+        players[0] = await srv.createPlayer(this.pseudo)
```

Comment:
I already saw this in a PR from @yannis-mlgrn , creating the player only if he doesn't exists and this PR was merged.

Reply:
Yeah I think I helped him with this part or we just wrote the same thing, but there was no conflict when rebasing